src/16-the-washington-post/index.html

```diff
@@ -0,0 +1,94 @@
+                    <div class="absolute-element">
+                        <span class="placeholder-text">Email address</span>
+                        <span class="btn-clear" onClick="clearInput()">
+                            <img src="./img/clear-icon.png" alt="">
+                        </span>
+                    </div>
+                </div>
+                <button type="submit" class="btn-next franklin-font">Next</button>
+                <div class="btn-sign-link franklin-font">Email a sign in link</div>
+            </form>
+            <div class="or-legend">
+                <div class="vertical-line"></div>
+                <span class="or-text">or</span>
+                <div class="vertical-line"></div>
+            </div>
+            <p class="description-text">To connect a sign in method, the email must match the one on your Washington Post account.</p>
+            <div class="other-sign">
+                <div class="other-sign-list">
+                    <div class="icon-sign" style="margin-top: 5px">
+                        <img src="./img/amazon.svg" alt="">
+                    </div>
+                    <div class="title-sign franklin-font">Continue with Amazon</div>
+                </div>
+                <div class="other-sign-list">
+                    <div class="icon-sign" style="margin-top: 3px">
+                        <img src="./img/facebook.png" alt="" style="width: 19px;">
+                    </div>
+                    <div class="title-sign franklin-font">Continue with Facebook</div>
+                </div>
+                <div class="other-sign-list">
+                    <div class="icon-sign" style="margin-top: 3px">
+                        <img src="./img/google.png" alt="" style="width: 18px;">
+                    </div>
+                    <div class="title-sign franklin-font">Continue with Google</div>
+                </div>
+                <div class="other-sign-list">
+                    <div class="icon-sign">
+                        <img src="./img/apple-md.png" alt="">
+                    </div>
+                    <div class="title-sign franklin-font">Continue with Apple</div>
+                </div>
+            </div>
+            <span class="register franklin-font">Need an account? <a href="">Sign up</a></span>
+        </div>
+    </section>
+
+
+    <!-- ADDITIONAL CODE -- start YOU MIGHT NOT NEEDED -->
+    <footer class="franklin-font">
+        <a href="/login-page-css/"> Back to Home </a>
+        <span class="author">
+          Crafted with ❤️ by <a href="https://github.com/tentpoles" rel="noopener noreferrer" target="_blank">Bhimo Al Kautsar</a> © 2021
+        </span>
+        <a href="https://github.com/mazipan/login-page-css" rel="noopener noreferrer" target="_blank"> Github Repo </a>
+    </footer>
+    <!-- ADDITIONAL CODE -- end YOU MIGHT NOT NEEDED -->
+    <script>
+        const clearInput = () => {
```

Comment:
use `var` instead, also do not use arrow function here

Reply:
I'll be back soon